scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
`echo("  Usage: bash $0 <major|minor|patch>");` should be `node`.  Other than that, the once over looks good.